server/routerlicious/kubernetes/routerlicious/templates/fluid-configmap.yaml

```diff
@@ -202,6 +202,7 @@ data:
         "usage": {
             "clientConnectivityCountingEnabled": "{{ .Values.usage.clientConnectivityCountingEnabled }}",
             "signalUsageCountingEnabled": {{ .Values.usage.signalUsageCountingEnabled }}
+            "httpUsageCountingEnabled": {{ .Values.usage.httpUsageCountingEnabled }}
```

Comment:
@nisha-singh can you open a follow up PR to add a comma at the end of the previous line? Without it this is resulting in malformed JSON and our (FF) internal deployment in AKS got into a crash loop :)